production.ini

```diff
@@ -19,6 +19,8 @@ stablekarma_comment = This update has reached the stable karma threshold and wil
 testing_approval_msg_based_on_karma = This update has reached the stable karma threshold and can be pushed to stable now if the maintainer wishes.
 not_yet_tested_msg_based_on_karma = This update has not reached the stable karma threshold.
 
+disable_automatic_push_to_stable = Negative karma received, disabling automatic push to stable. Please push manually if you decide the issue is not severe.
```

Comment:
I suggest a slightly different wording. Perhaps something like:

Bodhi is disabling automatic push to stable due to negative karma. The maintainer may manually if they determine that the issue is not severe.

This is at your option.